v21.2/bulk-delete-data.md

```diff
@@ -140,25 +140,71 @@ CockroachDB records the timestamp of each row created in a table in the `crdb_in
 
 CockroachDB does not support Time to Live (TTL) on table rows. To delete "expired" rows, we recommend automating a batch delete process using a job scheduler like `cron`.
 
-For example, suppose that every morning you want to delete all rows in the [`tpcc`](cockroach-workload.html#tpcc-workload) `history` table that are older than a month. To do this, you could use the example Python script that [batch deletes on the non-indexed `h_date` column](#batch-delete-on-a-non-indexed-column).
+For example, suppose that every morning you want to delete all rows in the [`rides` table](movr.html#the-movr-database) in the [`movr` database](movr.html) that are older than a month. To do this, you could write a Python script that batch-deletes rows based on the values of an indexed [`TIMESTAMP`](timestamp.html) column, and then run the script with a daily `cron` job.
 
-To run the script with a daily `cron` job:
+1. To record the last day and time a row was updated, create a `TIMESTAMP` column with an [`ON UPDATE` expression](add-column.html):
```

Comment:
nit: should this and below be called TIMESTAMPTZ

Reply:
Done.